core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -811,6 +811,11 @@ public enum Property {
           + " that begin with 'table.file' can be used here. For example, to set the compression"
           + " of the sorted recovery files to snappy use 'tserver.wal.sort.file.compress.type=snappy'.",
       "2.1.0"),
+  TSERV_FAILED_BULK_COPY_THREADS("tserver.failed.bulk.threads", "2", PropertyType.COUNT,
```

Comment:
Digging around I found that code that uses this in main is dead code.  Followed the constant Constants.ZBULK_FAILED_COPYQ in 2.1 and main and found that nothing adds to the queue in main.  This queue was used by bulk v1 to copy failed files.  Bulk v2 does not copy failed.

So in main we need to remove Constants.ZBULK_FAILED_COPYQ and any code that uses it which would include the code in the tserver that uses this new prop.

With the dead code in main being the only thing that would use this prop, probably do not want to introduce this new prop.  May make sense to remove the dead code in main first so that inform what to do in 2.1

Reply:
I updated this in f2a7a64 to remove the new property and just deprecate the unused property in main. I dropped some TODOs to remove some of the things that were apparent. I also moved some code around in LogSorter.